prboom2/src/v_video.h

```diff
@@ -299,6 +299,13 @@ void V_FreePlaypal(void);
 // [XA] get number of palettes in the current playpal
 int V_GetPlaypalCount(void);
 
+typedef struct ColorEntry_s
+{
+  int r, g, b;
+} ColorEntry_t;
+
+ColorEntry_t V_GetPatchColor (int lumpnum);
+
```

Comment:
PalEntry_s and SDL_Color already exist for exactly this.
Id prefer if SDL_Color was used everywhere.